UPGRADE.asciidoc

```diff
@@ -1776,6 +1776,13 @@ The realms are now multi-tenant aware, in order to upgrade your configuration to
 
   /usr/local/pf/addons/upgrade/to-8.2-realm-conf.pl
 
+
+Upgrade sudoers file (All linux distributions)
```

Comment:
all Linux

Reply:
@extrafu : in that case, I will update many line in that file (see 'Database schema update' titles).